keyboards/KeyHive/Maypad/Maypad.c

```diff
@@ -0,0 +1,51 @@
+/* Copyright 2019 codybender
```

Comment:
The keyboard and vendor directories should be all lowercase, to avoid problems with case-insensitive OSes.